components/header.tsx

```diff
@@ -6,20 +6,36 @@ import InnerHeader from './innerHeader';
 export const TabIds = {
     HOME: 'home',
     PRODUCTS: 'products',
+    ORDERS: 'orders',
 };
 
 export const TabRoutes = {
     [TabIds.HOME]: '/',
     [TabIds.PRODUCTS]: '/products',
+    [TabIds.ORDERS]: '/orders',
 };
 
+export const RouteTitles = {
+    [TabIds.HOME]: 'Home',
+    [TabIds.PRODUCTS]: 'Products',
+    [TabIds.ORDERS]: 'Orders',
+};
+
+const HeaderlessRoutes = [
+    '/orders/[orderId]',
```

Comment:
this will never be hit since you've defined it in `InnerRoutes` and that will be hit first (in the conditional inside of useEffect)